doc/02-Installation.md.d/From-Source.md

```diff
@@ -0,0 +1,22 @@
+# Installing Icinga Notifications Web from Source
+
+Please see the Icinga Web documentation on
+[how to install modules](https://icinga.com/docs/icinga-web-2/latest/doc/08-Modules/#installation) from source.
+Make sure you use `notifications` as the module name. The following requirements must also be met.
+
+### Requirements
+
+- PHP (≥7.2)
+- PHP needs the following extensions to be installed and activated:
+    - `cURL`
+    - `dom`
+    - `json`
+    - `libxml`
+- [MySQL](https://www.php.net/manual/en/ref.pdo-mysql.php)
+  or [PostgreSQL](https://www.php.net/manual/en/ref.pdo-pgsql.php) PDO PHP libraries
+- [Icinga Notifications Daemon](https://github.com/Icinga/icinga-notifications)
```

Comment:
`Icinga Notifications` only please

Reply:
They use daemon to differentiate between the golang and web part over at https://github.com/Icinga/icinga-notifications/pull/210.
I do agree that it might seem superflous, but just keeping it Icinga Notifications and Icinga Notifications Web might seem more confusing to and enduser.

Reply:
Then they're doing it wrong as well. We use product names in documentation, and `Icinga DB` isn't called `Icinga DB Daemon` for a reason. So it's `Icinga Notifications`.

Reply:
Just talked with them. Mentioning `daemon` (lowercase) is fine as part of a sentence where `Icinga Notifications` is referenced. However, with `Icinga Notifications Daemon` it's part of the name which is incorrect.

Reply:
Would this be fitting for this exact documentation paragraph then:
[Icinga Notifications](https://github.com/Icinga/icinga-notifications) (daemon) or would you just rather only mention the product name?

Reply:
In this case, only the product name